File: app/src/main/java/com/pitchai/flickrrecentphotobrowser/presentation/view/GridPhotoImageView.java
package com.pitchai.flickrrecentphotobrowser.presentation.view;

import android.graphics.Bitmap;

/**
 * Created by pitchairajamani on 3/23/17.
 */

public interface GridPhotoImageView {
    void renderPhotoImageView(Bitmap bitmap);
    void renderError(String Error);
}

File: app/src/main/java/com/pitchai/flickrrecentphotobrowser/presentation/view/GridPhotoImageView.kt
package com.pitchai.flickrrecentphotobrowser.presentation.view

import android.graphics.Bitmap

/**
 * Created by pitchairajamani on 3/23/17.
 */
interface GridPhotoImageView {
    fun renderPhotoImageView(bitmap: Bitmap?)
    fun renderError(Error: String?)
}